avocado/core/sysinfo.py

```diff
@@ -187,6 +187,11 @@ class Daemon(Command):
     :param compress_logf: Whether to compress the output of the command.
     """
 
+    def __init__(self, cmd, logf=None, compress_log=False):
```

Comment:
@PraveenPenguin Since the parent class constructor parameters aren't relevant in this class, I would suggest to keep API this way:
```python
def __init__(self, *args, **kwargs):
    super().__init__(*args, **kwargs)
```
So if the parent class adds new constructor arguments there's no need to update anything here and the API is still valid. 

What do you think @apahim?

Reply:
No strong opinion. Particularly I'd prefer the `*args`/`**kwargs`, even though that would make this class the only one in this module doing that.

Reply:
@cacarrara ah good point i will adopt this